Documentation/policy/intro.rst

```diff
@@ -96,7 +114,7 @@ egress are omitted, the rule has no effect.
 ----
 
 endpointSelector
-  Selects the endpoints to which the policy rules contained must be applied to.
+  Selects the endpoints which the policy rules contained must be applied to.
```

Comment:
Nit: Selects the endpoints to which the policy rules contained must be applied.

Reply:
This is a weird sentence in general.

Reply:
I plan to rewrite the paragraph, please take a look when I push that out.